### ISSUE TITLE
 Add a module to stress fallback queue using agent simulator 

### ISSUE THREAD

**Issue by pacoqack** [wazuh/wazuh-qa#1172]
Related to https://github.com/wazuh/wazuh-qa/issues/1168

We need to add an agent_simulator submodule to stress the fallback queue, that's, the queue that handles all the event that fall in the else clause in this switch statement: https://github.com/wazuh/wazuh/blob/2a96afbbafae5e4086c8938f431487aae0297a42/src/analysisd/analysisd.c#L1184-L1393

- [x] Find an example of a valid log typically sent to this queue.
- [x] Add a new way to generate different valid messages for the queue.
- [x] Add the new submodule for the agent simulator.

**Comment by pacoqack** [comment]
We will use SSH logs to stress this queue:

```
{"timestamp":"2021-03-24T10:12:23.786+0000","agent":{"id":"000","name":"centos8"},"manager":{"name":"centos8"},"id":"1616580743.3150094","full_log":"Mar 24 10:12:21 centos8 sshd[12241]: Connection closed by authenticating user root 172.17.1.1 port 56546 [preauth]","predecoder":{"program_name":"sshd","timestamp":"Mar 24 10:12:21","hostname":"centos8"},"decoder":{"parent":"sshd","name":"sshd"},"location":"/var/log/secure"}
{"timestamp":"2021-03-24T10:12:33.796+0000","agent":{"id":"000","name":"centos8"},"manager":{"name":"centos8"},"id":"1616580753.3150094","full_log":"Mar 24 10:12:32 centos8 sshd[12247]: Connection closed by authenticating user root 172.17.1.1 port 56548 [preauth]","predecoder":{"program_name":"sshd","timestamp":"Mar 24 10:12:32","hostname":"centos8"},"decoder":{"parent":"sshd","name":"sshd"},"location":"/var/log/secure"}
{"timestamp":"2021-03-24T10:12:37.800+0000","rule":{"level":5,"description":"sshd: Attempt to login using a non-existent user","id":"5710","mitre":{"id":["T1110"],"tactic":["Credential Access"],"technique":["Brute Force"]},"firedtimes":1,"mail":false,"groups":["syslog","sshd","invalid_login","authentication_failed"],"pci_dss":["10.2.4","10.2.5","10.6.1"],"gpg13":["7.1"],"gdpr":["IV_35.7.d","IV_32.2"],"hipaa":["164.312.b"],"nist_800_53":["AU.14","AC.7","AU.6"],"tsc":["CC6.1","CC6.8","CC7.2","CC7.3"]},"agent":{"id":"000","name":"centos8"},"manager":{"name":"centos8"},"id":"1616580757.3150341","full_log":"Mar 24 10:12:36 centos8 sshd[12249]: Invalid user dashdiashds from 172.17.1.1 port 56550","predecoder":{"program_name":"sshd","timestamp":"Mar 24 10:12:36","hostname":"centos8"},"decoder":{"parent":"sshd","name":"sshd"},"data":{"srcip":"172.17.1.1","srcport":"56550","srcuser":"dashdiashds"},"location":"/var/log/secure"}
{"timestamp":"2021-03-24T10:12:37.800+0000","agent":{"id":"000","name":"centos8"},"manager":{"name":"centos8"},"id":"1616580757.3150341","full_log":"Mar 24 10:12:36 centos8 sshd[12249]: Connection closed by invalid user dashdiashds 172.17.1.1 port 56550 [preauth]","predecoder":{"program_name":"sshd","timestamp":"Mar 24 10:12:36","hostname":"centos8"},"decoder":{"parent":"sshd","name":"sshd"},"location":"/var/log/secure"}
```

Example of generated alert:

```

** Alert 1616580757.3150341: - syslog,sshd,invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_10.6.1,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_AU.6,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2021 Mar 24 10:12:37 centos8->/var/log/secure
Rule: 5710 (level 5) -> 'sshd: Attempt to login using a non-existent user'
Src IP: 172.17.1.1
Src Port: 56550
Mar 24 10:12:36 centos8 sshd[12249]: Invalid user dashdiashds from 172.17.1.1 port 56550
```

**Comment by pacoqack** [comment]
It works:
![image](https://user-images.githubusercontent.com/15269938/112295018-c64dca00-8c93-11eb-960c-050aa04bed3e.png)
